openapi/v2/support-notifications.yaml

```diff
@@ -823,17 +800,17 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/ErrorResponse'
-  /notification/slug/{slug}:
+  /notification/id/{id}:
```

Comment:
Should we also have By Subscription Name? i.e. /notification/subscription/name/{name}?

Reply:
Added.